app/config/sculpin_site.yml

```diff
@@ -19,6 +19,9 @@ subspecies:
     confoo:
         common: "ConFoo"
         latin: "fooianum"
+    darkmira:
+        common: "Darkmira PHP Tour"
```

Comment:
DarkmiraTour PHP.

This is a event name.

Reply:
What is the name by which people will refer to the elePHPant?  Will it match the conference name exactly?

Reply:
@cookieguru Yes, it has the same conference name.

Reply:
@alessandrofeitoza Thanks, I've updated it